### DISCUSSION TITLE
How to hide information messages?

### DISCUSSION THREAD

**Discussion by rektosaure** [photobooth-app/photobooth-app#362] [Q&A]
![Capture d’écran 2024-10-14 à 16 29 32](https://github.com/user-attachments/assets/0d30aa7e-d42a-46a4-a86a-1e81e521d384)

How to hide this kind of message on display?

Thanks

**Comment by mgineer85** [comment]
Can't be hidden currently. It's just one time on startup/refresh. If it's shown more often probably there is a connection issue?

**Comment by rektosaure** [comment]
All share and print actions also appear.
This looks more like a debug message than a useful message for the common user I think.
